pkgs/development/web/postman/default.nix

```diff
@@ -0,0 +1,97 @@
+{ stdenv, lib, gnome2, fetchurl, pkgs, xlibs, udev, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "postman-${version}";
+  version = "5.5.2";
+
+  src = fetchurl {
+    url = "https://dl.pstmn.io/download/version/${version}/linux64";
+    sha1 = "68886197A8375E860AB880547838FEFC9E12FC64";
+    name = "${name}.tar.gz";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  buildPhase = ":";   # nothing to build
+
+  installPhase = ''
+    mkdir -p $out/share/postman
+    cp -R * $out/share/postman
+    mkdir -p $out/bin
+    mkdir -p $out/lib
+    ln -s $out/share/postman/Postman $out/bin/postman
+
+    # fix the path in the desktop file
+    #substituteInPlace \
+    #  $out/share/applications/masterpdfeditor4.desktop \
+    #  --replace /opt/ $out/opt/
+  '';
+
+  preFixup = let
+    libPath = lib.makeLibraryPath [
+      stdenv.cc.cc.lib
```

Comment:
Add `pkgs.udev.lib` here and the you should be able to remove the `libudev.so.1` hack below.

Reply:
Done. Thanks for this one :-)